packages/cactus-test-tooling/src/main/typescript/fabric/fabric-test-ledger-v1.ts

```diff
@@ -323,6 +370,815 @@ export class FabricTestLedgerV1 implements ITestLedger {
+      const couchdbFile = await this.populateFile(
+        addOrgXDirectoryPath,
+        "couch",
+        orgName,
+        couchDbPort,
+        dockerOutputAddOrgXDirectoryPath,
+      );
+      const caFile = await this.populateFile(
+        addOrgXDirectoryPath,
+        "ca",
+        orgName,
+        caPort,
+        dockerOutputAddOrgXDirectoryPath,
+      );
+
+      const composeFile = await this.populateFile(
+        addOrgXDirectoryPath,
+        "compose",
+        orgName,
+        peerPort,
+        dockerOutputAddOrgXDirectoryPath,
+      );
+
+      const cryptoFile = await this.populateFile(
+        addOrgXDirectoryPath,
+        "crypto",
+        orgName,
+        peerPort,
+        addOrgXDirectoryPath,
+      );
+
+      const configTxGenFile = await this.populateFile(
+        addOrgXDirectoryPath,
+        "configTxGen",
+        orgName,
+        peerPort,
+        addOrgXDirectoryPath,
+      );
+
+      // Pass add-Org files
+
+      const sourceFiles = [
+        couchdbFile,
+        caFile,
+        composeFile,
+        cryptoFile,
+        configTxGenFile,
+      ];
+
+      // Code taken from Fabric's Connector
+      Checks.truthy(sourceFiles, `${fnTag}:sourceFiles`);
+      Checks.truthy(Array.isArray(sourceFiles), `${fnTag}:sourceFiles array`);
+
+      /*
+      for (const sourceFile of sourceFiles) {
+        const { filename, filepath, body } = sourceFile;
+        const relativePath = filepath || "./";
+        const subDirPath = path.join(tmpDirPath, relativePath);
+        fs.mkdirSync(subDirPath, { recursive: true });
+        const localFilePath = path.join(subDirPath, filename);
+        fs.writeFileSync(localFilePath, body, "base64");
+      }
+      */
+      const remoteDirPath = path.join("/", "add-org-" + orgName);
+      log.debug(`SCP from/to %o => %o`, addOrgXDirectoryPath, remoteDirPath);
+      await ssh.putDirectory(addOrgXDirectoryPath, remoteDirPath);
+      log.debug(`SCP OK %o`, remoteDirPath);
+
+      log.debug(`Initializing docker commands`);
+
+      const envVarsList = {
+        VERBOSE: "true",
+        PATH: "$PATH:/fabric-samples/bin",
+        GO111MODULE: "on",
+        FABRIC_CFG_PATH: `/add-org-${orgName}`,
+        FABRIC_LOGGING_SPEC: "DEBUG",
+        CHANNEL_NAME: channel,
+        CLI_TIMEOUT: 300,
+        CLI_DELAY: 15,
+        MAX_RETRY: 5,
+        ORG_NAME: orgName,
+        ORG_MSPID: mspId,
+        DATABASE: database,
+        CA: certificateAuthority,
+        NEW_ORG_PORT: peerPort,
+        COUCH_DB_PORT: couchDbPort,
+        CA_PORT: caPort,
+        CONFIG_TX_GEN_PATH: path.join(remoteDirPath),
+        COMPOSE_FILE_COUCH: path.join(
+          remoteDirPath,
+          "docker",
+          couchdbFile.filename,
+        ),
+        COMPOSE_FILE: path.join(remoteDirPath, "docker", composeFile.filename),
+        COMPOSE_FILEe_CA: path.join(remoteDirPath, "docker", caFile.filename),
+        //COMPOSE_PROJECT_NAME: "cactusfabrictestnetwork_test",
```

Comment:
@RafaelAPB Recommending to remove the comments if they are just dead-ends (same considerations apply as above)

```suggestion
```